lib/debug/breakpoint.rb

```diff
@@ -8,9 +8,13 @@ class Breakpoint
 
     attr_reader :key
 
-    def initialize do_enable = true
+    def initialize cond:, command:, path:, do_enable: true
```

Comment:
could you use optional keywords?

Reply:
Since all types of breakpoints should support those options by default, I think asking subclasses to explicitly pass them is a way to prevent similar issues in the future.

Reply:
if it is mandatory, please use mandatory parameters.